elytron/src/test/java/org/wildfly/extension/elytron/DomainTestCase.java

```diff
@@ -269,13 +270,19 @@ public void testPermissionMappers() throws Exception {
         Roles roles = Roles.fromSet(new HashSet<>(Arrays.asList(new String[]{"role1", "role2"})));
 
         serviceName = Capabilities.PERMISSION_MAPPER_RUNTIME_CAPABILITY.getCapabilityServiceName("SimplePermissionMapperRole");
-        PermissionMapper mapper = (PermissionMapper) services.getContainer().getService(serviceName).getValue();
+        ServiceController<PermissionMapper> permissionMapperServiceController = (ServiceController<PermissionMapper>) services.getContainer().getService(serviceName);
+        permissionMapperServiceController.setMode(ServiceController.Mode.ACTIVE);
```

Comment:
@Mergers: This is a bit odd; in the subsystem configuration for this test, SimplePermissionMapperRole and SimplePermissionMapperPrincipal are declared but are not configured with any security domain. Before this change, both services that back them were in ACTIVE mode, but now they are LAZY. Since nobody is using them in the xml subsystem configuration, I had to start them manually here to make the test pass. I'm not sure if it is the correct approach.